### ISSUE TITLE
Unable to use array of custom types

### ISSUE THREAD

**Issue by meizon** [bijoutrouvaille/fireward#16]
If I create the following:

```
type Session = {
  dayOfWeek: number
}

type Booking = {
  sessions: Session[]
}
```

The rules that get generated incorrectly check that sessions is of type Session and not an array of Session.

It works as expected if I use string[] or number[]

**Comment by bijoutrouvaille** [comment]
Bug confirmed. Thank you for the report. I should have some time tonight to work on it.

**Comment by bijoutrouvaille** [comment]
Should be fixed in 1.4.1, which is now available on NPM and other official channels.

**Comment by meizon** [comment]
Thank you for your rapid work on this! I will check it out.
